geode-core/src/test/java/org/apache/geode/management/internal/cli/commands/GemfireDataCommandsDUnitTest.java

```diff
@@ -15,14 +15,14 @@
 
 package org.apache.geode.management.internal.cli.commands;
 
+import static org.apache.geode.distributed.ConfigurationProperties.LOG_LEVEL;
```

Comment:
Just a general comment on this test: maybe it's a good idea to separate this big test into smaller tests that covers one command per test, like RebalanceCommandDunitTest, QueryCommandDUnitTest etc. This way, we can do some common setup across tests and use ClassRule to start up the cluster instead of Rule.

Reply:
I only just saw this comment, but we did that! See the commit here: https://github.com/apache/geode/pull/2125/commits/b092914b548122651b38d920fff3092b0a1afa52